codespell_lib/data/dictionary.txt

```diff
@@ -3339,21 +3355,25 @@ defferent->different, deferent,
 defferential->differential, deferential,
 defferently->differently
 deffering->differing
+defferred->deferred
 deffers->differs, defers,
 deffine->define
 deffined->defined
 deffinition->definition
 deffinitively->definitively
 deffirent->different
+defice->device
 defien->define
 defiend->defined
 defin->define
 definance->defiance
 definate->definite
 definately->definitely
+defination->definition
 definatly->definitely
 defind->defined, defund,
 definded->defined, defunded,
+define'd->defined
```

Comment:
Can we handle non-alphanumerics by default? I've had two false positives where someone has written `#define'd` in a comment, which while technically a misspelling, is also entirely intentional. I'm ignoring for now, but I'm just wondering if this ever caught genuine typos?

Reply:
I'm happy with that solution

Reply:
https://github.com/codespell-project/codespell/pull/629